apps/web/src/app/platform/version.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from "@angular/core";
+import { catchError, firstValueFrom, map } from "rxjs";
+
+import { PlatformUtilsService } from "@bitwarden/common/platform/abstractions/platform-utils.service";
+import { SdkService } from "@bitwarden/common/platform/abstractions/sdk/sdk.service";
+
+type Version = {
+  client: string;
+  sdk: string;
+};
+
+@Injectable({
+  providedIn: "root",
+})
+export class VersionService {
+  constructor(
+    private platformUtilsService: PlatformUtilsService,
+    private sdkService: SdkService,
+  ) {}
+
+  init() {
```

Comment:
I prefer a more specific name for methods like this.

```suggestion
  applyVersionToWindow() {
```